app/javascript/lib/vue_toasts.ts

```diff
@@ -3,7 +3,7 @@ import { POSITION, useToast } from 'vue-toastification';
 import 'css/vue_toastification_with_overrides.scss';
 
 const DEFAULT_VUE_TOASTIFICATION_OPTIONS = Object.freeze({
-  position: POSITION.TOP_CENTER,
+  position: POSITION.TOP_RIGHT,
```

Comment:
Top center is often kind of annoying and blocks content. Let's make the default top right, where I think it's less likely that there will be valuable content.